Sources/Networking/Core/APIManaging.swift

```diff
@@ -11,10 +11,10 @@ import Foundation
 // MARK: - Defines API managing
 
 /// A definition of an API layer with methods for handling API requests.
-public protocol APIManaging {
+public protocol APIManaging: URLSessionInvalidatable {
```

Comment:
just a philosophical question. Do we need to APIManaging to conform this or we can leave it just to APIManager itself?